PsfRuntime/CreateProcessHook.cpp

```diff
@@ -153,9 +155,17 @@ BOOL WINAPI CreateProcessFixup(
         processInformation = &pi;
     }
 
+    // Redirect createProcess arguments if any in native app data to per user per app data
+    CharT* cnvtCmdLine = new CharT[MAX_CMDLINE_PATH];
+    if (cnvtCmdLine)

```

Comment:
if this is command line input, it can't exceed 8191, why reserving extra? else changes the define string

Reply:
This is commandLine argument in createProcess win API call whose maximum lenggth can be 32,767 characters
https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa